lib/client.js

```diff
@@ -61,7 +61,7 @@ Client.prototype.setup = function(){
 Client.prototype.connect = function(name){
   debug('connecting to namespace %s', name);
   if (!this.server.nsps[name]) {
-    this.packet({ type: parser.ERROR, nsp: name, data : 'Invalid namespace'});
+    this.packet({ type: parser.ERROR, nsp: name, data : 'Invalid namespace'}, false, false, true);
```

Comment:
I think a packet should have these options, so that we can make it isomorphic with the client.

https://github.com/Automattic/socket.io-client/pull/805/files#diff-9b2f90c89d460ee80ced18e01748824eR139